aws/etc/packer/tools/python/stardog/cluster/start_stardog.py

```diff
@@ -0,0 +1,13 @@
+import stardog.cluster.utils as utils
```

Comment:
Do we really need python wrappers for this and stop_stardog.py?

Reply:
Probably not, I can remove those and retest